pysatMadrigal/instruments/madrigal_dst.py

```diff
@@ -67,7 +67,8 @@
 madrigal_inst_code = 212
 madrigal_tag = {'': {'': "30006"}}
 madrigal_start = dt.datetime(1957, 1, 1)
-madrigal_end = dt.datetime.utcnow()
+madrigal_end = pysat.utils.time.filter_datetime_input(
```

Comment:
Drive-by quick comment: 
I'd recommend adding your rationale for filtering the date time to get one without a timezone, or with a timezone...